task_geo/data_sources/hdx_oxfd/hdx_oxfd_formatter.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from datetime import datetime
+
+import pandas as pd
+
+from task_geo.common.country_codes import iso2_to_iso3, iso3_to_iso2
+
+logger = logging.getLogger(__name__)
+
+
+def hdx_oxfd_formatter(raw, start_date, end_date, country_iso2):
+    """Formats raw pandas.DataFrame
+    - Drops columns ending with _note
+    - Orders Columns
+    - Converts ISO3 to ISO2 country codes
+
+    Arguments:
+        raw (pandas.DataFrame): from hdx_acap_connector
+
+    Returns: pandas.DataFrame
+
+    """
+    data = raw.copy()
+    data.columns = [column.lower() for column in data.columns]
+    # Remove columns with notes
+    drop_columns = [column for column in data.columns if column.find("_note") > -1]
+    data = data.drop(drop_columns, axis=1)
+    # Filter by country(s)
+    iso3 = []
+    for c in country_iso2:
+        iso3.append(iso2_to_iso3(c))
+    data = filter_by_country(data, iso3)
+    # Filter by date
+    data = filter_with_date(data, start_date, end_date)
+    # Convert ISO3 Country code to ISO2 Country Code
+    if len(data) > 0:
+        data['country_iso2'] = data.apply(lambda x: iso3_to_iso2(x['countrycode']), axis=1)
+    else:
+        return data
+
+    # Rename Column
+    data.rename(columns={"countryname": "country"}, inplace=True)
+    # Reorder Columns
+    column_order = ['country', 'country_iso2', 'date', 's1_school closing',
+                    's1_isgeneral', 's2_workplace closing', 's2_isgeneral',
+                    's3_cancel public events', 's3_isgeneral', 's4_close public transport',
+                    's4_isgeneral', 's5_public information campaigns', 's5_isgeneral',
+                    's6_restrictions on internal movement', 's6_isgeneral',
+                    's7_international travel controls', 's8_fiscal measures',
+                    's9_monetary measures', 's10_emergency investment in health care',
+                    's11_investment in vaccines', 's12_testing framework',
+                    's13_contact tracing', 'confirmedcases', 'confirmeddeaths',
+                    'stringencyindex', 'stringencyindexfordisplay']
```

Comment:
Avoid hanging indent, it's more useful to put them in a new indentation order than to continue where the list creation started, will leave you with more characters till the max-line

Reply:
How do I put it in a new indentation order? Can you give an example? I usually go with what the ide auto-indents.

Reply:
Something like:

my_multi_line_list = [
   'one_element', ......
   'more_elements', ....
]